Project.toml

```diff
@@ -24,7 +24,7 @@ AWS = "1.63.1"
 ArrowTypes = "1.2"
 Compat = "3.29.0"
 EzXML = "0.9, 1"
-FilePathsBase = "=0.9.11, =0.9.12, =0.9.13, =0.9.14, =0.9.15"
+FilePathsBase = "=0.9.11, =0.9.12, =0.9.13, =0.9.14, =0.9.15, =0.9.18"
```

Comment:
```suggestion
FilePathsBase = "0.9"
```

Reply:
Ok, wasn't aware that there has also been another no-op parse been implemented. Just needed a quickfix on my side since the upgrade of CSV broke my production code... ;)

Reply:
the issue is that FilePathsBase 0.9.16 started relying on `parse` in `readdir` (ref https://github.com/rofinn/FilePathsBase.jl/pull/152) in a way that conflicted with https://github.com/JuliaCloud/AWSS3.jl/pull/129, so to temporarily solve the issue, we compat bounded strictly to 0.9.11-0.9.15 (ref https://github.com/JuliaCloud/AWSS3.jl/pull/232). But if we add the `parse` method here, then we don't need that strict compat, so we can get new FilePathsBase releases that have fixes like the `readavailable` one.